x/evm/blocktest/config.go

```diff
@@ -0,0 +1,6 @@
+package blocktest
+
+type Config struct {
```

Comment:
need json declarations and parser like https://github.com/sei-protocol/sei-chain/blob/seiv2/x/evm/replay/config.go#L8-L45